<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkVTKScalarsToColorsWidget</class>
 <widget class="QWidget" name="ctkVTKScalarsToColorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScalarsToColorsWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="PointIdLabel">
       <property name="toolTip">
        <string>Point ID</string>
       </property>
       <property name="text">
        <string>Point:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="PointIdSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Point ID</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkColorPickerButton" name="ColorPickerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="displayColorName">
        <bool>false</bool>
       </property>
       <property name="dialogOptions">
        <set>ctkColorPickerButton::UseCTKColorDialog</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="XLabel">
       <property name="toolTip">
        <string>X coordinate of the current point</string>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="XSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>X coordinate of the current point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="OpacityLabel">
       <property name="toolTip">
        <string>Opacity of the current point</string>
       </property>
       <property name="text">
        <string>O:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="OpacitySpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Opacity of the current point</string>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MidPointLabel">
       <property name="toolTip">
        <string>Position of the midpoint</string>
       </property>
       <property name="text">
        <string>M:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="MidPointSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Position of the midpoint</string>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SharpnessLabel">
       <property name="toolTip">
        <string>Sharpness of the midpoint</string>
       </property>
       <property name="text">
        <string>S:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="SharpnessSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Sharpness of the midpoint</string>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkMoreButton" name="HideShowButton">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="ctkVTKScalarsToColorsView" name="View"/>
   </item>
   <item row="2" column="0">
    <widget class="ctkDoubleRangeSlider" name="XRangeSlider">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="maximumValue">
      <double>1.000000000000000</double>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkDoubleRangeSlider" name="YRangeSlider">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkVTKScalarsToColorsView</class>
   <extends>QWidget</extends>
   <header>ctkVTKScalarsToColorsView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleRangeSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMoreButton</class>
   <extends>QPushButton</extends>
   <header location="global">ctkMoreButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
